src/test/java/org/mockitousage/stacktrace/ClickableStackTracesTest.java

```diff
@@ -34,8 +35,7 @@ public void shouldShowActualAndExpectedWhenArgumentsAreDifferent() {
             verifyTheMock(1, "not foo");
             fail();
         } catch (ArgumentsAreDifferent e) {
-            assertContains("callMethodOnMock(", e.getMessage());
-            assertContains("verifyTheMock(", e.getMessage());
+            assertThat(e).hasMessageContaining("callMethodOnMock(").hasMessageContaining("verifyTheMock(");
```

Comment:
To keep consistency the second `hasMessageContaining` should be in the another line.